libsawtooth/src/client/mod.rs

```diff
@@ -59,6 +59,13 @@ pub trait SawtoothClient {
         batch_ids: Vec<&str>,
         wait: Option<Duration>,
     ) -> Result<Option<Vec<Status>>, SawtoothClientError>;
+    /// Send one or more batches to the validator.
```

Comment:
This doc comment should include some details about what the arguments do, since their names don't really tell us that. This format is pretty good for detailing method args: https://github.com/Cargill/splinter/blob/master/libsplinter/src/admin/store/mod.rs#L391

Reply:
Fixed doc comment to include arguments

Reply:
Fixed, included argument definitions in doc comment